src/mixins/ui/lasso-tool-button-svg.js

```diff
@@ -0,0 +1,52 @@
+export const buttonSvg = buttonType => {
+  let svgContent
+
+  switch (buttonType) {
+    case "circle":
+      svgContent = `<?xml version="1.0" encoding="utf-8"?><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 48 48" style="enable-background:new 0 0 48 48;" xml:space="preserve"><style type="text/css">.st0{fill:none;stroke-width:4;stroke-miterlimit:10;stroke-dasharray:8,3;}</style><circle class="st0" cx="24" cy="24" r="16"/></svg>`
```

Comment:
Remove generic "Layer_1" id; assuming this just came along with the figma

Reply:
Interestingly enough, that actually came from an online base64 -> svg converter, so strange; will remove and ensure it doesn't effect anything, good catch

Reply:
@cjvil Ok removed and nothing broke :)

Reply:
@cmatzenbach Nice! Did you see my first comment?

Reply:
@cjvil Sorry...I keep forgetting about this one lol.  So the issue is this corresponds with https://github.com/heavyai/immerse/pull/9302 (`matzy/FE-17793_tint-mode-colors-update` Immerse branch).  This PR is turning those icons into svg, which are then styled through immerse.  Should've mentioned that before, my fault.  Once approved, this won't get merged in until that Immerse PR is ready, since without those styles it does break it.  Let me know if that all makes sense.  Here's a screencast showing how it looks with the corresponding immerse changes:

https://github.com/heavyai/heavyai-charting/assets/15268289/b3138d97-6a4e-41cc-ab9b-e9805175aaf3

Reply:
Gotcha sounds good, I would still add defaults to charting to make the icons look sane when charts used outside of Immerse.